project01/dominik/snake.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SNAKE_H
+#define SNAKE_H
+
+#include <vector>
+#include <cinder/app/RendererGl.h>
```

Comment:
Order of includes: from the most to the least specific, i.e. your includes, then libraries, then STL